packages/universal-components/src/WithHover/index.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export { default as withHover } from './withHover';
```

Comment:
Can you please also add export to main `src/index.js`.

Reply:
Oops, I forgot that 🙈